WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -52,12 +52,12 @@ enum DashboardCard: String, CaseIterable {
         }
     }
 
-    func shouldShow(for blog: Blog) -> Bool {
+    func shouldShow(for blog: Blog, mySiteSettings: MySiteSettings = MySiteSettings()) -> Bool {
         switch self {
         case .quickActions:
             return true
         case .quickStart:
-            return QuickStartTourGuide.shouldShowChecklist(for: blog)
+            return QuickStartTourGuide.shouldShowChecklist(for: blog) && mySiteSettings.defaultSection() == .dashboard
```

Comment:
My first urge was to extract that to a `static` func inside `QuickStartTourGuide`, something like: `QuickStartTourGuide.shouldShowChecklistOnDashboard(for: blog)`

But at the same time I think being very explicit here about the default section might be interesting for future readers, so probably best to leave as it is.